slidez-core/src/main/java/com/binarystudio/academy/slidez/infrastructure/security/auth/AuthController.java

```diff
@@ -51,7 +52,15 @@
 		if (authResponse.isEmpty()) {
 			return new ResponseEntity<>("Incorrect password or user email.", HttpStatus.UNAUTHORIZED);
 		}
-		return new ResponseEntity<>(authResponse.get(), HttpStatus.OK);
+		return new ResponseEntity(authResponse.get(), HttpStatus.OK);
+	}
+
+	@PostMapping("login-by-token")
+	public ResponseEntity<AuthResponse> loginByToken(
+			@RequestBody AuthorizationByTokenRequest authorizationByTokenRequest) {
+		Optional<AuthResponse> authResponseOptional = this.authService.performLoginByToken(authorizationByTokenRequest);
+		return authResponseOptional.map(authResponse -> new ResponseEntity<>(authResponse, HttpStatus.OK))
+				.orElseGet(() -> new ResponseEntity<>(HttpStatus.UNAUTHORIZED));
```

Comment:
Why I see here google auth??

Reply:
It only looks like google auth, but it uses different service

Reply:
Oauth* 
You can see in commits that it is here

Reply:
I got that just now, sorry